docker-bake.hcl

```
@@ -308,6 +316,19 @@ target "scilus-fsl" {
     output = ["type=docker"]
 }
 
+target "scilus-freesurfer" {
+    inherits = ["freesurfer"]
+    contexts = {
+        freesurfer-base = "target:scilus-base"
+    }
+    cache-from = [
+        "type=registry,ref=${dockerhub-user-pull}/build-cache:scilus-deps",
+        "type=registry,ref=scilus/build-cache:scilus-deps"
+    ]
+    tags = notequal("", DEPS_TAG) ? stamp_tag("scilus/scilus-deps:${DEPS_TAG}", timestamp()) : []
```

```suggestion
    tags = notequal("", DEPS_TAG) ? stamp_tag("scilus/scilus-freesurfer:${DEPS_TAG}", timestamp()) : []
```

let's put this in a separate dockerhub repository.